### ISSUE TITLE
Tag picker has invalid aria-attribute value

### ISSUE THREAD

**Issue by shyama1208** [OfficeDev/office-ui-fabric-react#5042]
TagPicker has invalid ARIA attribute value (aria-controls) causing Keros failure.
 
![image](https://user-images.githubusercontent.com/26054709/40766863-85e87068-64ce-11e8-862b-b5d5dc2fe1e6.png)

Repro: 
Run Keros for Chrome for Tag picker.
You can use the below link .
https://developer.microsoft.com/en-us/fabric#/components/pickers

Simply run keros on the above page.
